### ISSUE TITLE
Cannot pull latest Rocket.Chat

### ISSUE THREAD

**Issue by k0nsl** [RocketChat/Rocket.Chat#14327]
I was going to update my instance to `1.0.0` today but I cannot seem to pull it because I keep getting the following response:

```
root@goychat01:~/rocket.chat# ./update-rocketchat.sh
1.0.0: Pulling from rocketchat/rocket.chat
bc9ab73e5b14: Already exists 
3f117e0d3349: Already exists 
29f0971335c1: Already exists 
a717aa7e730b: Already exists 
0a2d97db0648: Already exists 
8b2475665ab0: Already exists 
3bcb2b74efb5: Extracting [=============>                                     ]  43.45MB/165.6MB
unexpected EOF
```

I do not have much to go on, but perhaps somebody else have run into this and is able to clear it up. I can't seem to come up with a solution to it.

**Comment by k0nsl** [comment]
This may also be of help:

```
0a2d97db0648: Already exists
8b2475665ab0: Already exists
3bcb2b74efb5: Extracting [======================>                            ]  74.65MB/165.6MB
Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "compose/cli/main.py", line 57, in main
  File "compose/cli/main.py", line 108, in perform_command
  File "compose/cli/main.py", line 728, in up
  File "compose/project.py", line 377, in up
  File "compose/service.py", line 293, in ensure_image_exists
  File "compose/service.py", line 791, in pull
  File "compose/progress_stream.py", line 18, in stream_output
  File "compose/utils.py", line 52, in split_buffer
  File "compose/utils.py", line 28, in stream_as_text
  File ".tox/py27/lib/python2.7/site-packages/docker/client.py", line 225, in _stream_helper
  File ".tox/py27/lib/python2.7/site-packages/requests/packages/urllib3/response.py", line 271, in read
requests.packages.urllib3.exceptions.ProtocolError: ('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))
docker-compose returned -1
root@goychat01:~/rocket.chat# 
```

**Comment by geekgonecrazy** [comment]
uh oh.. seems docker is having some issues.  What does your command look like inside your update-rocketchat.sh ?

Also make sure 1.0.2 to get bug fixes.  :)

**Comment by k0nsl** [comment]
Hi @geekgonecrazy !

I tried to pull `1.0.2` today with the same result as shown above.

At any rate, here is some relevant information which may be of help.
```

root@goychat01:~# docker version
Client:
 Version:           18.06.3-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        d7080c1
 Built:             Wed Feb 20 02:28:55 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.3-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       d7080c1
  Built:            Wed Feb 20 02:27:21 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

```
root@goychat01:~# docker-compose -v
docker-compose version 1.24.0, build 0aa59064
root@goychat01:~#
```

And the contents of `update-rocketchat.sh` is quite lame, see:
cat: ./rocketchat/update-rocketchat.sh: No such file or directory
```
root@goychat01:~# cat ./rocket.chat/update-rocketchat.sh
#service docker restart
docker pull rocketchat/rocket.chat:1.0.2
docker-compose stop chat
docker-compose rm chat
docker-compose up -d chat
root@goychat01:~#
```

My `docker-compose.yml` is pretty basic as well:
```
root@goychat01:~# cat ./rocket.chat/docker-compose.yml
mongo:
  image: mongo
# volumes:
#    - ./data/runtime/db:/data/db
#    - ./data/dump:/dump
  command: mongod --smallfiles --oplogSize 128
  restart: always

chat:
  image: rocketchat/rocket.chat:1.0.2
#  image: rocketchat/rocket.chat:0.54.2
# volumes:
#    - ./uploads:/app/uploads
  environment:
    - PORT=3000
    - ROOT_URL=https://k0nsl.org
    - MONGO_URL=mongodb://mongo:27017/rocketchat
  links:
    - mongo:mongo
  ports:
    - 3000:3000
  restart: always

# hubot, the popular chatbot (add the bot user first and change the password before starting this image)
#hubot:
#  image: rocketchat/hubot-rocketchat
#  environment:
#    - ROCKETCHAT_URL=rocketchat:3000
#    - ROCKETCHAT_ROOM=GENERAL
#    - ROCKETCHAT_USER=bot
#    - ROCKETCHAT_PASSWORD=botpassword
#    - BOT_NAME=bot
# you can add more scripts as you'd like here, they need to be installable by npm
#    - EXTERNAL_SCRIPTS=hubot-help,hubot-seen,hubot-links,hubot-diagnostics
#  links:
#    - chat:chat
# this is used to expose the hubot port for notifications on the host on port 3001, e.g. for hubot-jenkins-notifier
#  ports:
#    - 3001:8080
root@goychat01:~#
```

That's it. Not a lot to go on, unfortunately. The issue just popped up out of nowhere the other day and now the chat is ofline as a result of this.

**Comment by geekgonecrazy** [comment]
@k0nsl if it never pulled i'd revert back to the previous version.

Then try doing docker pull until docker lets you pull it.  Then once its downloaded successfully you can try upgrading.

But yeah nothing wrong with your setup there that I can see.  Seems docker isn't liking something

**Comment by geekgonecrazy** [comment]
Did you manage to figure this out?

**Comment by KevLehman** [comment]
This pull request has been inactive for a while, so we’re closing it to keep the backlog manageable.
If you’d like to continue working on it, feel free to reopen or submit a new PR. Thanks for the contribution!
